revolution/motor.py

```diff
@@ -42,10 +44,12 @@ def _control(self) -> None:
                 acceleration_input = max(
                     contexts.motor_acceleration_pedal_input,
                     contexts.motor_acceleration_paddle_input,
+                    contexts.motor_acceleration_cruise_control_input
```

Comment:
max only accepts 2 arguments or a single iterable. This is an error. Please do something like...

```Python
max(
    (
        item0,
        item1,
        item2,
    ),
)
```